index.html

```diff
@@ -17,10 +17,54 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container">
+        <h1>Code Journal</h1>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div data-view="entry-form">
+        <form>
+          <h2>New Entry</h2>
+          <div class="row">
+            <img
+              src="images/placeholder-image-square.jpg"
+              alt="placeholder-image-square"
+              class="column-full column-half" />
```

Comment:
You should usually always wrap your `img`'s in a `div`. You should wrap this in a `div` and apply the column classes to the `div` instead.

Don't forget to give your `img` `width: 100%` after doing this